database/queries.sql

```diff
@@ -0,0 +1,37 @@
+--Fetch all the favourite items for a given user
+SELECT i.name
+FROM onboarding_schema.users as u, onboarding_schema.mark_as_favourite as fav, onboarding_schema.items i
+Where u.name = 'John Doe' and u.id = fav."user" and fav.item = i.id
+
+--Fetch all users together with their favourite items, even if the user doesn't have any items
+SELECT u.name, i.name AS favorite_item
+FROM (onboarding_schema.users AS u LEFT JOIN onboarding_schema.mark_as_favourite AS fav ON u.id = fav."user") LEFT JOIN onboarding_schema.items AS i ON fav.item = i.id
+ORDER BY u.name;
```

Comment:
I would add u.id to the select in case we have two users with same name

Also you could add a `GROUP BY u.id, u.name` and change `i.name` in the `SELECT` to be `STRING_AGG(i.name, ', ')`
This way for each unique user you can get a list of their favourite items